test/services/optout/OptOutServiceSpec.scala

```diff
@@ -353,4 +353,52 @@ class OptOutServiceSpec extends UnitSpec
       }
     }
   }
+  "OptOutService.optOutCheckPointPageViewModel" when {
+    val CY = TaxYear.forYearEnd(2024)
+    val PY = CY.addYears(-1)
```

Comment:
why not use CY.**previousYear** method? optional comment since this is test code

Reply:
didn't knew that existed. will implement.